src/tokens/eth/0x8E870D67F660D95d5be530380D0eC0bd388289E1.json

```diff
@@ -1,11 +1,11 @@
 {
-  "symbol": "PAX",
-  "name": "Paxos Standard",
+  "symbol": "USDP",
+  "name": "Pax Dollar",
   "type": "ERC20",
   "address": "0x8E870D67F660D95d5be530380D0eC0bd388289E1",
   "ens_address": "",
   "decimals": 18,
-  "website": "https://www.paxos.com/standard/",
+  "website": "https://www.paxos.com/usdp/",
```

Comment:
we might as well fill in the social information etc below as well. you can use the links in [etherscan](https://etherscan.io/token/0x8e870d67f660d95d5be530380d0ec0bd388289e1)

Reply:
I noticed you did originally but it was removed in subsequent commits. I'll add it back.

Reply:
ah interesting let's leave it out then. maybe they don't use it

Reply:
A few seem to use it but most don't. A quick grep for `facebook` shows 124 non-empty and 5569 empty, and similar for `twitter`.